sensing/autoware_pointcloud_preprocessor/schema/vector_map_inside_area_filter_node.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Vector Map Inside Area Filter Node",
+  "type": "object",
+  "definitions": {
+    "vector_map_inside_area_filter": {
+      "type": "object",
+      "properties": {
+        "polygon_type": {
+          "type": "string",
+          "description": "polygon type to be filtered",
+          "default": "no_obstacle_segmentation_area"
+        },
+        "use_z_filter": {
+          "type": "boolean",
+          "description": "use z value for filtering",
+          "default": "false"
+        },
+        "z_threshold": {
```

Comment:
Boundaries might be useful here

Reply:
@amadeuszsz 
Thanks for the comment.
Just take a quick look at the code. It seems that z_threshold can be both positive and negative. Thus, I am not sure whether a boundary is useful here.